src/scene/shader-lib/programs/lit-shader.js

```diff
@@ -1239,6 +1239,8 @@ class LitShader {
                         }
                     } else if (light._shadowType === SHADOW_PCF5) {
                         shadowReadMode = "PCF5x5";
+                    } else if (light._shadowType === SHADOW_PCF1) {
+                        shadowReadMode = "PCF1x1";
```

Comment:
Kinda wish this was a switch statement TBH. But that's not your fault of course! 😄

Reply:
I've switched this over.